WordPress/Classes/ViewRelated/Reader/ReaderPostMenu.swift

```diff
@@ -131,14 +148,17 @@ open class ReaderPostMenu {
         let siteID = post.siteID
         let toFollow = !post.isFollowing
 
+        let postService = ReaderPostService(managedObjectContext: post.managedObjectContext!)
```

Comment:
I'd probably unwrap this with a `guard` rather than force unwrapping – just in case!